src/Components/RelatedProducts/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/no-access-state-in-setstate */
```

Comment:
a basic functioning carousel component which displays up to 4 ProductCards in one row